examples/most-redis-docker/README.md

```diff
@@ -0,0 +1,132 @@
+# Most + Redis + Docker
+
+A suite of simple apps to illustrate how to use most to handle redis events.
+
+This suite may be run using [docker](https://www.docker.com) or [node.js
+10.x](https://nodejs.org/en/).  If you have docker installed already or would
+like to try it, follow the docker instructions.  Otherwise, you will need to be
+running redis locally.  Follow the node instructions to install node and redis.
+
+## Install
+
+### Pre-requisites
+
+1. Install [git](https://help.github.com/articles/set-up-git/)
+2. Clone this repo:
+
+```sh
+git clone git@github.com:mostjs/core.git
+cd examples/most-redis-docker
+```
+
+### Assumptions
+
+The remainder of this document assumes your working directory is
+`most-redis-docker`.  You may have to modify some of the following directions
+if you chose another folder name.
+
+This project uses environment variables.  If you are using the windows command
+prompt, you may have to change some commands slightly to use env vars. For
+example, rather than type
+`REDIS_MYCHANNEL=most-redis-docker-example npm run consume`, you would type
+`set REDIS_MYCHANNEL=most-redis-docker-example&&npm run consume`.
+
+### Option 1: Docker
+
+1.  Install [docker](https://www.docker.com/get-started).  (You will need to
+    create a [DockerHub](https://hub.docker.com) account.)
+2.  Build the docker image for this project:
+
+```sh
+docker build -t most-redis-docker .
+```
+
+### Option 2: node.js
+
+1. Install [node 10.x](https://nodejs.org/en/) or the latest LTS version.
+2. Install the dependencies for this project:
+
+```sh
+npm install
+```
+
+3. Install [redis](https://redis.io/topics/quickstart).
+
+If you already have an instance of redis running, you may edit the docker-
+compose.yml file in the `most-redis-docker` folder.  Change any values for
+`REDIS_HOST` and `REDIS_PORT` according to your redis instance
+
+## Run
+
+When running this example, the publish service periodically sends JSON messages
+to redis, and the consume services receive the JSON messages and transform
+them.  In your terminal(s), you should expect to see informational logs from
+each service as they start up and as they process the JSON messages.
+
+### Option 1: Docker
+
+1.  From the `most-redis-docker` folder, create and run the docker-compose
+    network defined in `docker-compose.yml`:
+
+```sh
+docker-compose up
+```
+
+Use `Ctrl-C` to stop the docker-compose network.
+
+You may also use `docker-compose up -d` and `docker-compose down` to start and
+stop the docker-compose network in the background.
+
+### Option 2: node.js
+
+1.  Start redis.  See the [redis docs](https://redis.io/topics/quickstart) for
+    troubleshooting and options.
+
+```sh
+redis-server
+```
+
+2. From the `most-redis-docker` folder:
+
+```sh
+REDIS_MYCHANNEL=most-redis-docker-example node consume
```

Comment:
I just walked through the both the docker and non-docker instructions.  Docker worked perfectly.  Well done!

In the non-docker instructions, I found something I hadn't noticed before: The instructions use both names: `REDIS_MYCHANNEL`, and `REDIS_CHANNEL`.  The code wants `REDIS_CHANNEL`.  So, I think all we need to do is use update the docs to use `REDIS_CHANNEL` everywhere, and this whole thing is ready to 🚢.

<img width="558" alt="screen shot 2019-02-28 at 6 46 47 am" src="https://user-images.githubusercontent.com/90518/53564659-7f795e80-3b25-11e9-8a62-f3140f19c571.png">

<img width="486" alt="screen shot 2019-02-28 at 6 52 44 am" src="https://user-images.githubusercontent.com/90518/53564668-856f3f80-3b25-11e9-8b0d-a8c1a26c8d2e.png">

UPDATE: The `env` in my screenshots is because I use [fish](https://fishshell.com) rather than a bash-compatible shell.  Keeping the the instructions optimized for bash-compatible is :+1:, as I imagine most fish users are accustomed to adapting.

Reply:
Thanks.  I never would have noticed that!  Fixed.